terraform/warehouse/vcl/main.vcl

```diff
@@ -308,6 +326,12 @@ sub vcl_error {
         set obj.http.Content-Type = "text/html; charset=UTF-8";
         synthetic {"<html><head><title>301 Moved Permanently</title></head><body><center><h1>301 Moved Permanently</h1></center></body></html>"};
         return(deliver);
+    } else if (obj.status == 751) {
+        set obj.status = 308;
```

Comment:
are we doing 308s on the current service?

Reply:
```
❯ curl -XPOST https://pypi.python.org/ -i
HTTP/2 308
server: Varnish
retry-after: 0
location: https://pypi.org/
content-type: text/html; charset=UTF-8
accept-ranges: bytes
date: Mon, 30 Apr 2018 22:58:42 GMT
via: 1.1 varnish
x-served-by: cache-ewr18139-EWR
x-cache: HIT
x-cache-hits: 0
x-frame-options: deny
x-xss-protection: 1; mode=block
x-content-type-options: nosniff
x-permitted-cross-domain-policies: none
content-length: 124

<html><head><title>308 Permanent Redirect</title></head><body><center><h1>308 Permanent Redirect</h1></center></body></html>
```